### ISSUE TITLE
Feature request: adding SMA archive ?

### ISSUE THREAD

**Issue by esoPanda** [astropy/astroquery#727]
Could you consider adding [SMA archive](https://www.cfa.harvard.edu/cgi-bin/sma/smaarch.pl) ?

**Comment by bsipocz** [comment]
Yes, it was already suggested here: https://github.com/astropy/astroquery/issues/226

Are you interested in contributing it to astroquery? There is a template module here: http://astroquery.readthedocs.io/en/latest/template.html

**Comment by esoPanda** [comment]
I wish I could ^_^
Never tried this kind of coding before. I may ask someone from SMA.

**Comment by Phantsure** [comment]
I would like to work on this, but I couldn't find coordinates for which there is data available on SMA. If anyone could guide.

**Comment by keflavich** [comment]
The link provided at the top of this Issue is all the information we have.  Anything else we want to do we have to reverse engineer from that site.

**Comment by Phantsure** [comment]
There is a RAW file related to every observation and that is quite large(in GBs) so there should be an option to download the file from object, right?
